db/seeds.rb

```diff
@@ -6,10 +6,35 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-ArtType.create(name: "visual art")
-ArtType.create(name: "writing")
-ArtType.create(name: "music")
-ArtType.create(name: "dance")
-ArtType.create(name: "theater")
-ArtType.create(name: "filmmaking")
-ArtType.create(name: "photography")
+ArtType.find_or_create_by(name: "visual art")
+ArtType.find_or_create_by(name: "writing")
+ArtType.find_or_create_by(name: "music")
+ArtType.find_or_create_by(name: "dance")
+ArtType.find_or_create_by(name: "theater")
+ArtType.find_or_create_by(name: "filmmaking")
+
+50.times do |n|
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.